my-code/src/pages/Home/styles.js

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+import variables from '../../styles/variables';
+
+export const Container = styled.div`
+  ul.pagination {
```

Comment:
Is it really necessary for the `ul` selector? The code seems "dirty" with this kind of selector.

Reply:
Was used because I thought it would provide more meaning. If someone adds another `ul` in the container in the future, the browser will apply the pagination formatting to the new `ul` also (even if a class is provided). Agreed or do you still think it isn't necessary?